### PR TITLE
Add lint CI for Python samples

### PR DESCRIPTION
**Pull request by jhuleatt**
Use [Black](https://black.readthedocs.io/en/stable/index.html) to lint Python samples

### CODE REVIEW

File: .github/workflows/test_python.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+env:
+  CI: true
+
+jobs:
+  unit:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+          - "3.11"
```
Comment (kevinthecheung) [inline .github/workflows/test_python.yml:35]: 3.10 too?

File: Python/alerts-to-discord/main.py [hunk 2]
```diff
@@ -58,9 +60,10 @@ def post_message_to_discord(bot_name: str, message_body: str,
 # [START v2CrashlyticsAlertTrigger]
 @crashlytics_fn.on_new_fatal_issue_published(secrets=["DISCORD_WEBHOOK_URL"])
 def post_fatal_issue_to_discord(
-        event: crashlytics_fn.CrashlyticsNewFatalIssueEvent) -> None:
+    event: crashlytics_fn.CrashlyticsNewFatalIssueEvent,
+) -> None:
     """Publishes a message to Discord whenever a new Crashlytics fatal issue occurs."""
-# [END v2CrashlyticsAlertTrigger]
+    # [END v2CrashlyticsAlertTrigger]
```
Comment (kevinthecheung) [inline Python/alerts-to-discord/main.py:66]: i think it needs to be col 0, throughout

File: Python/taskqueues-backup-images/main.py [hunk 3]
```diff
@@ -31,8 +31,7 @@
 BACKUP_START_DATE = datetime(1995, 6, 17)
 BACKUP_COUNT = params.IntParam("BACKUP_COUNT", default=100).value()
 HOURLY_BATCH_SIZE = params.IntParam("HOURLY_BATCH_SIZE", default=600).value()
-BACKUP_BUCKET = params.StringParam("BACKUP_BUCKET",
-                                   input=params.ResourceInput).value()
+BACKUP_BUCKET = params.StringParam("BACKUP_BUCKET", input=params.ResourceInput).value()
```
Comment (kevinthecheung) [inline Python/taskqueues-backup-images/main.py:34]: :(

**Review by kevinthecheung — CHANGES_REQUESTED** [review]

File: Python/alerts-to-discord/main.py (hunk 2 above)
Comment (jhuleatt) [reply Python/alerts-to-discord/main.py:66]: Per our conversation, looks like indented tags are ok

File: Python/taskqueues-backup-images/main.py (hunk 3 above)
Comment (jhuleatt) [reply Python/taskqueues-backup-images/main.py:34]: I can shorten the line length to prevent this. Will give it a go

Comment (jhuleatt) [reply Python/taskqueues-backup-images/main.py]: looks like it wrapped it, but differently. LGTY?

File: .github/workflows/test_python.yml (hunk 1 above)
Comment (jhuleatt) [reply .github/workflows/test_python.yml:36]: done!

File: Python/pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[project]
+name = "functions_samples"
+version = "1.0.0"
+[tool.black]
+line-length = 80
```
Comment (jhuleatt) [inline Python/pyproject.toml:19]: @kevinthecheung I set line length to 80

**Review by kevinthecheung — APPROVED** [review]

**Pull request merged**